xml/templates/message_templates/event_offline_receipt_html.tpl

```diff
@@ -173,10 +173,10 @@
         {if $isShowLineItems}
           {foreach from=$participants key=index item=currentParticipant}
             {if $isPrimary || {participant.id} === $currentParticipant.id}
-              {if $isPrimary && $lineItems|@count GT 1} {* Header for multi participant registration cases. *}
+              {if $isPrimary && ($participants|@count > 1)} {* Header for multi participant registration cases. *}
```

Comment:
@larssandergreen out of scope for this PR but does this use of  `|@count`  work with Smarty3?

Reply:
@eileenmcnaughton Yes, in Smarty 2,  |count would output 1 1 1 1 and |@count would output 4, but in Smarty 3 both output 4. So |@count is compatible with both, |count works differently.

Reply:
@larssandergreen ok  what is recommended for Smarty 4 - do you know?

Reply:
@eileenmcnaughton no change as far as I know.